### PR TITLE
Added the feature engineering with Feast tutorial

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+feast -c feature_repo materialize-incremental "2024-07-24T08:00:00"
+----
+
+. Running the command above created an IMap that corresponds to the "user_transaction_count_7d" feature in the Hazelcasst cluster.
+You can list it using the following command:
++
+[source,shell]
+----
+clc object list map
+----
```
Comment (rebekah-lawrence) [inline docs/modules/integrate/pages/feature-engineering-with-feast.adoc:308]:
```suggestion
----
+
```

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+. Check the contents of the feature IMap to check the data written by Feast:
++
+[source,shell]
+----
+clc map -n feast_batch_user_transaction_count_7d entry-set | head -10
+----
+
+== Summary
+
+In this tutorial, you learned how to setup a feature engineering project that uses Hazelcast as the online store and PostgreSQL as the offline store..
```
Comment (rebekah-lawrence) [inline docs/modules/integrate/pages/feature-engineering-with-feast.adoc:330]:
```suggestion
In this tutorial, you learned how to set up a feature engineering project that uses Hazelcast as the online store and PostgreSQL as the offline store..
```

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+You also learned how to write Jet jobs that transform data and store it in a PostgreSQL table to be used by the Feast offline store.
+
+== See Also
+
+There is more to feature engineering with Hazelcast.
+
+Check out our documentation on Feast at: TODO lINK.
+
+If you have any questions, suggestions, or feedback please do not hesitate to reach out to us via https://slack.hazelcast.com/[Hazelcast Community Slack].
```
Comment (rebekah-lawrence) [inline docs/modules/integrate/pages/feature-engineering-with-feast.adoc:339]:
```suggestion
If you have any questions, suggestions, or feedback please do not hesitate to reach out to us through https://slack.hazelcast.com/[Hazelcast Community Slack].
```

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]
Looking good, Yuce, very much appreciated 😀

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc [hunk 4]
```diff
@@ -0,0 +1,339 @@
+= Feature Engineering with Feast
+:description: This tutorial will get you started with feature engineering using the Hazelcast integration with Feast.
+
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/modules/integrate/pages/feature-engineering-with-feast.adoc:3]: Updated at: 0fbd59166130220da5208cc35582a26546185b92

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+run build_jet batch_features
+----
+
+. You can now create the Jet jobs and run them:
+[source,shell]
+----
+clc job submit build/jet/batch_features/libs/*.jar /home/hazelcast/data
+----
+
+. You can list the running jobs and verify that the jobs completed successfully using the following command:
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/modules/integrate/pages/feature-engineering-with-feast.adoc:271]: Updated at: 0fbd59166130220da5208cc35582a26546185b92

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc [hunk 6]
```diff
@@ -0,0 +1,339 @@
...
+[source,shell]
+----
+clc job list
+----
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/modules/integrate/pages/feature-engineering-with-feast.adoc:276]: Updated at: 0fbd59166130220da5208cc35582a26546185b92

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc [hunk 7]
```diff
@@ -0,0 +1,339 @@
...
+ 0c0d-c9a3-c143-0001 | N/A  | COMPLETED | 2024-07-24 19:15:09 | 2024-07-24 19:15:09
+ 0c0d-c9a3-c141-0001 | N/A  | COMPLETED | 2024-07-24 19:15:07 | 2024-07-24 19:15:07
+ 0c0d-c9a3-c140-0001 | N/A  | COMPLETED | 2024-07-24 19:15:05 | 2024-07-24 19:15:06
+----
+
+== Materialization
+
+Materialization is the process of transferring features from the offline store to the online store. In this case, from PostgreSQL to Hazelcast.
+
+. Run the following command to materialize the features:
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/modules/integrate/pages/feature-engineering-with-feast.adoc:299]: Updated at: 0fbd59166130220da5208cc35582a26546185b92

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc (hunk 1 above)
Comment (yuce) [reply docs/modules/integrate/pages/feature-engineering-with-feast.adoc:312]: Updated at: 0fbd59166130220da5208cc35582a26546185b92

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc [hunk 8]
```diff
@@ -0,0 +1,347 @@
...
+== Summary
+
+In this tutorial, you learned how to set up a feature engineering project that uses Hazelcast as the online store and PostgreSQL as the offline store..
+You also learned how to write Jet jobs that transform data and store it in a PostgreSQL table to be used by the Feast offline store.
+
+== See Also
+
+There is more to feature engineering with Hazelcast.
+
+Check out the our documentation about Feast:
```
Comment (rebekah-lawrence) [inline docs/modules/integrate/pages/feature-engineering-with-feast.adoc:342]:
```suggestion
Check out our documentation about Feast:
```

**Review by rebekah-lawrence — APPROVED** [review]
One tiny thing - otherwise LGTM, so approving. Thank you!

File: docs/modules/integrate/pages/feature-engineering-with-feast.adoc (hunk 8 above)
Comment (yuce) [reply docs/modules/integrate/pages/feature-engineering-with-feast.adoc]: Updated at: https://github.com/hazelcast/hz-docs/pull/1195/commits/932db89934f4adb68537f9d97cb610171d9d9b6e

**Pull request merged**